README.md

```diff
@@ -31,64 +31,117 @@ The PostgreSQL STIG provides guidance on the configuration of PostgreSQL to addr
 To run the PostgreSQL STIG Compliance Validator, there are specific requirements on both the database host as will as the STIG valudation host.
 
 #### Database Host
-- PostgreSQL 9.5+ cluster running on \*nix host
-- Remote access to PostgreSQL Server
-- lsof
-- netstat
+
+-   PostgreSQL 9.5+ cluster running on \*nix host
+-   Remote access to PostgreSQL Server
+-   lsof
+-   netstat
 
 #### STIG Validation Execution Host
-- Linux VM or Host
-- sudo access to install packages
+
+-   Linux VM or Host
+-   sudo access to install packages
 
 #### Required software on STIG Validation Execution Host
-- git
-- ssh
-- [InSpec](https://www.chef.io/products/chef-inspec/)
+
+-   git
+-   ssh
+-   [InSpec](https://www.chef.io/products/chef-inspec/)
 
 ### Setup Environment on STIG Validation Execution Host
+
 #### Install InSpec
-Goto https://downloads.chef.io/inspec/stable and copy download link
-> ex: https://packages.chef.io/files/stable/inspec/3.9.0/el/7/inspec-3.9.0-1.el7.x86_64.rpm
+
+Goto <https://downloads.chef.io/inspec/stable> and copy download link
+
+For example:
+
+-   <https://packages.chef.io/files/stable/inspec/3.9.0/el/7/inspec-3.9.0-1.el7.x86_64.rpm>
 
 ```sh
 sudo yum insall https://packages.chef.io/files/stable/inspec/3.9.0/el/7/inspec-3.9.0-1.el7.x86_64.rpm
 ```
 
-#### Ensure your InSpec version is at least 1.31.x
+#### Ensure your InSpec version is at least 3.x or higher
+
 ```sh
 inspec --version
 ```
 
-### Setting attributes.yml
+### Setting & Reviewing the Core Profile Attributes
+
+The `core` or `shared` attributes are set in the `inspec.yml` file in the profile. This stores the default values for the
+shared attributes that the profile uses to examine your system.
+
+These attributes _should not be edited directly_!
```

Comment:
I would make it

> These attributes _**should not be edited directly**_!

as this sounds pretty important 😄

Reply:
resolved